psi4/driver/procrouting/dft_funcs/dict_builder.py

```diff
@@ -120,27 +121,31 @@ def get_functional_aliases(functional_dict):
 
     # if the parent functional is already dispersion corrected, skip to next
     if "dispersion" in dict_functionals[functional_name]:
+        disp = dict_functionals[functional_name]['dispersion']
+        dashcoeff_supplement[disp['type']]['definitions'][functional_name] = disp['params']
+        # TODO still needs citation
+        # this is to "bless" dft_funcs dispersion definitions
```

Comment:
So the `func['dispersion']` only gets passed once, via `dashcoeff_supplement` (which is of the same stature, if present, as `intf_dftd3.dashcoeff` itself). This saves the param_tweaks role in `intf_dftd3.from_arrays(..., param_tweaks)` for partial user modifications to defined things, rather than having three sources of truth to shuffle through: literature (in dashcoeff), dict_builder, and user.

You've hit upon my unsolved case. :-) Yes, I think a disp-bearing fctl defined in input should be "custom". I have a test case, still making it work.